utils/not_doctested.txt

```diff
@@ -0,0 +1,135 @@
+docs/source/en/quicktour.md
```

Comment:
Files are added here if they're too big or heavy to run and test the code I believe, so we might be able to take `quicktour` out since its pretty light (and there are probably others)?

Reply:
Sure. Do we add these manually?

Reply:
Yes they have been added manually or now. We most likely will have to run this workflow at least once, to get an idea of how long some of the tests take and make changes accordingly. 

I've updated it so that the files here are either running training examples or they are translations of the english docs (since the code snippets are the same).